widget/slider.go

```diff
@@ -321,6 +387,11 @@ func (s *sliderRenderer) Layout(size fyne.Size) {
 
 	s.thumb.Move(thumbPos)
 	s.thumb.Resize(fyne.NewSize(diameter, diameter))
+
+	focusIndicatorSize := fyne.NewSize(theme.IconInlineSize()+theme.InnerPadding(), theme.IconInlineSize()+theme.InnerPadding())
```

Comment:
Maybe if this was coded relative to the calculated `diameter` (i.e. `diameter+theme.InnerPadding()`) the issue may be resolved? Just wondering as it could be a rounding error somewhere and that might work around it...

Reply:
I tried that initially but had some issues with getting the size to match the radio buttons. Will have a shot at it again though :)

Reply:
Tried a lot of things to work around the possible rounding error but to no avail (even tried using `theme.InnerPadding() + 1.75` which the `delta` variable becomes if simplified). I have no idea how to take this further with this issue unfortunately. I would appreciate some help if you have the time :sweat_smile:

Reply:
I believe that the code should be mostly fine in all other areas though. I have just added the remaining comments, moved some methods around to be in alphabetical order and fixed tap focus to work correctly.

Reply:
This looks to be solved now thanks to your slider wobble fix :)